File: hackertracker/src/main/java/com/shortstack/hackertracker/Network/SyncResponse.java
package com.shortstack.hackertracker.Network;

import com.google.gson.annotations.SerializedName;
import com.shortstack.hackertracker.Model.Item;

public class SyncResponse {

    @SerializedName("updateTime")
    public String time;
    @SerializedName("updateDate")
    public String date;

    public Item[] schedule;

    @Override
    public String toString() {
        return time + ", " + date + ", " + schedule.length;
    }
}

File: hackertracker/src/main/java/com/shortstack/hackertracker/Network/SyncResponse.kt
package com.shortstack.hackertracker.network

import com.google.gson.annotations.SerializedName
import com.shortstack.hackertracker.models.Event

data class SyncResponse(

        @SerializedName("update_date")
        val updatedDate: String,
        @SerializedName("schedule")
        val events: List<Event>

)
